rest/data/interface.go

```diff
@@ -92,14 +92,17 @@ type Connector interface {
 	///////////////
 	// Test Results
 	///////////////
-	// FindTestResultsByTaskId queries the database to find all test
+	// FindTestResultsByTaskID queries the database to find all test
 	// results with the given options. If the execution is not specified,
 	// this will return the test results from the most recent execution.
-	FindTestResultsByTaskId(context.Context, TestResultsOptions) ([]model.APITestResult, error)
+	FindTestResultsByTaskID(context.Context, TestResultsOptions) ([]model.APITestResult, error)
+	// FindTestResultsByDisplayTaskID queries the database to find all test
+	// results with the given options. If the execution is not specified,
+	// this will return the test results from the most recent execution.
+	FindTestResultsByDisplayTaskID(context.Context, TestResultsOptions) ([]model.APITestResult, error)
```

Comment:
Can't we just make one `FindTestResults` method for `ByTaskID` and `ByDisplayTaskID` since they have the exact same signature and the same code?